axelrod/tests/strategies/test_memoryone.py

```diff
@@ -222,11 +222,10 @@ def test_four_vector(self):
 
 class TestMemoryOnePlayer(unittest.TestCase):
 
-    def test_exception_if_four_vector_not_set(self):
+    def test_default_if_four_vector_not_set(self):
         player = MemoryOnePlayer()
-        opponent = axelrod.Player()
-        with self.assertRaises(ValueError):
-            player.strategy(opponent)
+        self.assertEqual(player._four_vector, {(C, C): 1.0, (C, D): 0.0,
+                                               (D, C): 0.0, (D, D): 1.0})
```

Comment:
Could we add a test catching the warning?

Reply:
Sure will look into it now!